contracts/integration-tests/contracts/test/StakingWithTokens.sol

```diff
@@ -0,0 +1,53 @@
+pragma solidity ^0.5.9;
+pragma experimental ABIEncoderV2;
+
+import "@0x/contracts-staking/contracts/src/Staking.sol";
+
+
+contract StakingWithTokens is
+    Staking
+{
+    address public wethAddress;
+    address public zrxVault;
+
+    bytes4 internal constant WETH_PROXY_ID = 0xf47261b0;
+
+    function setWethAddress(address weth)
+        public
+    {
+        wethAddress = weth;
+    }
+
+    function setZrxVault(address vault)
+        public
+    {
+        zrxVault = vault;
+    }
+
+    function getWethAssetData()
```

Comment:
Why do we need this function? Also, is there any reason we can't just import `TestStaking` from the staking package instead of using this contract?

Reply:
I originally included it here because this package needed something to be in the `contracts/` directory. I can remove this an add an empty contract until adding the dYdX contracts though.

Reply:
That should work, but on second thought, I will need to add functions for updating `weth` and `zrx`. It actually looks like the staking wrapper is setting these values in the staking logic contract rather than in the proxy, which is where we need the values for this test.

Reply:
I'll just update `TestStaking`.